scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2207,6 +2301,7 @@ static bool run_timer_cb(int key)
 struct layered_timer layered_timers[MAX_TIMERS] = {
 	{15LLU * NSEC_PER_SEC, CLOCK_BOOTTIME, 0},
 	{0LLU, CLOCK_BOOTTIME, 0},
+	{1LLU * NSEC_PER_SEC, CLOCK_BOOTTIME, 0},
```

Comment:
This changes the timer interval of the `NOOP_TIMER` and `MAX_TIMER` from the ordering of `layer_timer_callbacks`. I think the switch on 2289 needs to match the order of the `layered_timers`

Reply:
think the last commit fixes that